crates/omnix-ci/src/command/run.rs

```diff
@@ -220,6 +233,7 @@ pub async fn ci_run(
     run_cmd: &RunCommand,
     cfg: &OmConfig,
     nix_config: &NixConfig,
+    github_output: bool,
```

Comment:
You don't need this argument, because that information is already in the `run_cmd` argument.